src/base/bittorrent/torrentinfo.cpp

```diff
@@ -105,6 +81,11 @@ TorrentInfo &TorrentInfo::operator=(const TorrentInfo &other)
     return *this;
 }
 
+bool TorrentInfo::isValid() const
+{
+    return (m_nativeInfo != nullptr);
```

Comment:
what about using this? https://en.cppreference.com/w/cpp/memory/shared_ptr/operator_bool
`return static_cast<bool>(m_nativeInfo);`

Reply:
IMO, current form is more clear in this context.

Reply:
You might want reconsider since `operator!=` is removed in c++20.

Reply:
> You might want reconsider since `operator!=` is removed after c++20.

But it will still synthesized from `operator<=>`.

Reply:
ok.